sphinx/util/console.py

```diff
@@ -20,13 +47,15 @@
 
 _CSI = re.escape('\x1b[')  # 'ESC [': Control Sequence Introducer
 _ansi_re: re.Pattern[str] = re.compile(
-    _CSI + r"""
+    _CSI
+    + r"""
```

Comment:
Maybe this file could now be removed from the `ruff format` exclude list, as it looks like you formatted it manually?

Reply:
Well I created the current list with a little `Path.glob` use, I'm sure you can work out how to do the same, i.e to "expand" `sphinx/util/**/*`

Reply:
Ah, yes. I'll make another PR to expand everything (it's easier like that). I was wondering whether there was an option in ruff for that.